### PR TITLE
Add join section

### LINKED ISSUE #1075: Implement join program section
Size: M

### PR DESCRIPTION
**Pull request by techtolentino**
Fixes #1075

### CODE REVIEW

File: components/advocates/AdvocatesJoinSection.vue
Comment (korgan00) [inline components/advocates/AdvocatesJoinSection.vue:102]: The design has double space, you can set just `margin: $spacing-05;` because flex does not overlap margins.
To keep it in sync with `__access` you can put the same there.

File: components/advocates/AdvocatesJoinSection.vue
Comment (korgan00) [inline components/advocates/AdvocatesJoinSection.vue:59]: In very large, padding is not even on both sides, I think contained is the problem. If you change the `__feature` margin, this should have `padding: 0 $spacing-05;`  to match the design.

File: components/advocates/AdvocatesJoinSection.vue
Comment (korgan00) [inline components/advocates/AdvocatesJoinSection.vue:75]: You are using flex. I think it is better to use `flex` combined property or `flex-basis`. I suggest maybe something like `flex: 1 1 4 * $column-size-large;`
[flexbox guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

File: components/advocates/AdvocatesJoinSection.vue
Comment (korgan00) [inline components/advocates/AdvocatesJoinSection.vue:78]: Same flex things here but `flex: 1 1 45%;`

File: components/advocates/AdvocatesJoinSection.vue
Comment (korgan00) [inline components/advocates/AdvocatesJoinSection.vue:101]: Same flex things here

File: components/advocates/AdvocatesJoinSection.vue
Comment (korgan00) [inline components/advocates/AdvocatesJoinSection.vue:106]: Same flex things here

File: components/advocates/TheAdvocatesHeader.vue
Comment (korgan00) [inline components/advocates/TheAdvocatesHeader.vue:32]: This is not a join section change, so I think it is better to keep these changes in a separate PR.

**Review by korgan00 — CHANGES_REQUESTED** [review]

**Comment by korgan00** [comment]
Good work @techtolentino

File: components/advocates/AdvocatesJoinSection.vue
Comment (techtolentino) [reply components/advocates/AdvocatesJoinSection.vue:59]: @korgan00 - I think I'm missing what this looks like, can you please attach a screenshot :pray:

Comment (korgan00) [reply components/advocates/AdvocatesJoinSection.vue:59]: 2020-11-09 10_08_23-Window

Comment (techtolentino) [reply components/advocates/AdvocatesJoinSection.vue:59]: @korgan00 - I know that this `contained()` mixin was created long before I began on this project, and it seems like it's supposed to be that way?

@y4izus - can you confirm?

Comment (y4izus) [reply components/advocates/AdvocatesJoinSection.vue:59]: You both are right. On the one hand, yes, we use the `contained()`, but on the other hand, if I have understood Goyo properly, this `contained()` is making your code more complicated when managing the margins/paddings. If you add the `padding: 0 $spacing-05;` right before the `contained()`, you can easily align the items.

@korgan00 , that's what you were trying to explain, right?

Comment (techtolentino) [reply components/advocates/AdvocatesJoinSection.vue:59]: If I'm understanding @korgan00 correctly... the design shows `1rem` of left and right padding, but here, our container is showing `padding-left: 2.5rem; padding-right: 1rem`... and this is the problem, is that right?

The way I'm using the `contained` mixin on this page, I believe (correct me if I'm mistaken), is consistent with how it's used on other pages.

Comment (korgan00) [reply components/advocates/AdvocatesJoinSection.vue:59]: The design shows 2rem on each side. In your implementation, you have 1.5 from the container padding and 1 from the margin on the right side, and 2.5 from the container on the left.
That forces to make a difference between `__feature`  and `__access` margins and makes the CSS more complex.
If you apply this little change. You can merge the margins, flex, and media-queries from the `__feature`  and `__access`
```suggestion
    @include contained();
    padding: 0 1rem;
```

**Review by korgan00 — CHANGES_REQUESTED** [review]

**Review by eddybrando — APPROVED** [review]
LGMT!

But since I'm new, please wait for a _more authoritative_ approval before merging 😄

**Review by korgan00 — APPROVED** [review]
It is ok for me. 
Good job

**Review by y4izus — APPROVED** [review]
Great work and great reviews!! Thanks!!

**Pull request merged**
